web/modules/custom/helloworld/helloworld.info.yml

```diff
@@ -0,0 +1,10 @@
+name: 'HelloWorld'
+type: module
+description: 'A Custom Module for the Testing Purpose.'
+package: Custom
+core_version_requirement: ^9 || ^10
+dependencies:
+  - drupal:block
+  - drupal:views
+  - mymodule:mymodule
```

Comment:
Your module is currently not having any functionalities.
Yet you have added so many dependencies?

Reply:
I added them just for testing, if in future i get their need